lib/slide_hub/batch.rb

```diff
@@ -0,0 +1,100 @@
+class Batch
+  def self.execute
```

Comment:
__[rubocop]__
* Metrics/AbcSize: Assignment Branch Condition size for execute is too high. [37.55/30] [:ref](http://c2.com/cgi/wiki?AbcMetric)
* Metrics/MethodLength: Method has too many lines. [31/20] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)